src/checkers/inference/InferenceAnnotatedTypeFactory.java

```diff
@@ -572,6 +572,14 @@ protected InferenceViewpointAdapter createViewpointAdapter() {
             return Collections.singleton(vAnno);
         }
 
+        final List<? extends AnnotationMirror> annos = type.getAnnotationMirrors();
```

Comment:
This is to handle the special case of anonymous classes with explicit annotation, e.g.
`A a1 = new @OsUntrusted A() {};`
the declaration bound  of the anonymous class is the explicit annotation before the class identifier

Reply:
Could you add this as a comment?
Also add a comment what the then-block does, e.g. convert to the corresponding ConstantSlot. Is this the nicest way to do this?